homeassistant/components/devolo_home_control/exceptions.py

```diff
@@ -0,0 +1,5 @@
+"""Custom exceptions for the devolo_home_control integration."""
+
+
+class CredentialsInvalid(Exception):
```

Comment:
Inherit from HomeAssistantError.
```suggestion
class CredentialsInvalid(HomeAssistantError):
```